src/main/java/mu/semte/ch/api/kalliope/rest/AppController.java

```diff
@@ -41,6 +43,21 @@ public AppController(PersistService persistService) {
     return ResponseEntity.ok().build();
   }
   
+  @GetMapping(value = "/changes",
+  produces = "application/ld+json")
+  public ResponseEntity<String> change(
+  @RequestParam(required=false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime since, 
+  @RequestParam(required=false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime snapshot) {
+    
+    var dataset = persistService.getAllChanges(since, snapshot);
+    //TODO Refactor
+    StringWriter writer = new StringWriter();
+    RDFDataMgr.write(writer, dataset, Lang.JSONLD);
```

Comment:
you could maybe simply the writing by doing the following?

`ModelUtils.toString(dataset.getUnionModel(), Lang.JSONLD)`

except if there is a good reason not to do it like that? if yes this part could be extracted and put in mu-java (ModelUtils)

Reply:
I will indeed extract it to modelutils.

The reason is that the data is now in named graphs (e.g. inserts, deletes or consolidated) and some meta data is a separate default graph. This will now be more inline with the existing delta producers.

Reply:
Extraction will be done later today or tomorrow, hence the TODO. Main prio is to get this up and running asap.